### PR TITLE
Use hoard

### PR DESCRIPTION
**Pull request by eliasbenussi**

### CODE REVIEW

File: R/reports.R [hunk 1]
```diff
@@ -216,39 +216,15 @@ wait_and_check <- function(report_object) {
...
-
-  datasets_credentials <- get_datasets_credentials()
-
-  file_key <- paste0(
-    Sys.getenv("FACULTY_PROJECT_ID"),
-    report_object$active_version$report_path
-  )
-
-  # upload to s3
-  aws.s3::put_object(
```
Comment (acroz) [inline R/reports.R:245]: Is there any dependency of `rfaculty` on this AWS interface that can now be removed?

Comment (eliasbenussi) [reply R/reports.R:245]: Yes, good point I had forgotten about that

**Review by acroz — APPROVED** [review]
LGTM, let's merge assuming that:

- The tests have been run
- We've tested in in-place on a platform deployment

**Comment by eliasbenussi** [comment]
> * The tests have been run

I have successfully run the tests on a cube

> * We've tested in in-place on a platform deployment

Also done

**Review by mociarain — APPROVED** [review]

**Pull request merged**